pkg/kubelet/kubelet.go

```diff
@@ -124,8 +124,8 @@ const (
 	// Max amount of time to wait for the container runtime to come up.
 	maxWaitForContainerRuntime = 30 * time.Second
 
-	// Max amount of time to wait for node list/watch to initially sync
-	maxWaitForAPIServerSync = 10 * time.Second
+	// Period between checks for the node list/watch initial sync
+	waitForAPIServerSyncPeriod = 1 * time.Second
```

Comment:
Will this be too spammy? Do the requests reliably complete in 1s or do we risk asking again before they're done?

Reply:
nodeHasSynced() calls HasSynced() on an informer, so it just returns internal informer state without a per-call-request.

1s is the original nodeHasSynced() frequency.